docs/source/notes/autograd.rst

```diff
@@ -214,80 +214,250 @@ proper thread locking code to ensure the hooks are thread safe.
 .. _complex_autograd-doc:
 
 Autograd for Complex Numbers
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+----------------------------
 
-**What notion of complex derivative does PyTorch use?**
-*******************************************************
+tl;dr
+^^^^^
 
-PyTorch follows `JAX's <https://jax.readthedocs.io/en/latest/notebooks/autodiff_cookbook.html#Complex-numbers-and-differentiation>`_
-convention for autograd for Complex Numbers.
+- PyTorch Autograd returns gradients that are directly usable for optimization and gradient descent (no conjugation needed),
+  so existing optimizers work out of the box. We don’t make any attempt to support holomorphic functions specially;
+  in particular, derivatives are only well defined for real-valued loss functions.
```

Comment:
I realize this is my own damn fault because I wrote this text in the first place, but taken literally this text is wrong (I'm not sure what I even meant when I wrote this)

Reply:
hmmm I rephrased it. does it look any better?